README.md

```diff
@@ -1,2 +1,13 @@
-# review-workflow
-Demo how we're trying to do reviews in Guest Experience
+# Peer Review workflows
+
+How we do reviews in Guest Experience:
+
+1. Create a branch.
+2. Push it to GitHub.
+3. Create a pull-request as soon as you like.
+4. Do whatever you want with it. Squash and rebase to your heart's content. 
+5. Get reviewed.
+6. Just keep squashing and rebasing.
```

Comment:
Our current flow leaves us with a serious problem: once we rebase or squash or amend a given commit which has already been reviewed, it's impossible for the reviewer to see what actually changed since their review.

We could change the process so we don't ever change the commits which occurred prior to the peer review.